<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>SettingsDialog</class>
    <widget class="QDialog" name="SettingsDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>220</width>
                <height>171</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                    <item>
                        <widget class="QRadioButton" name="Adapter">
                            <property name="text">
                                <string>自适应大小</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <widget class="QRadioButton" name="radioButton_2">
                                    <property name="text">
                                        <string>固定大小</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
                                    <item>
                                        <widget class="QLabel" name="label">
                                            <property name="text">
                                                <string>横向</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QSpinBox" name="MaxXInput">
                                            <property name="maximum">
                                                <number>1000</number>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
                                    <item>
                                        <widget class="QLabel" name="label_2">
                                            <property name="text">
                                                <string>纵向</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QSpinBox" name="MinYInput">
                                            <property name="minimum">
                                                <number>-500</number>
                                            </property>
                                            <property name="maximum">
                                                <number>500</number>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QSpinBox" name="MaxYInput">
                                            <property name="minimum">
                                                <number>-500</number>
                                            </property>
                                            <property name="maximum">
                                                <number>500</number>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>SettingsDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>SettingsDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
